js/app.js

```diff
@@ -1,12 +1,47 @@
-;(function($){
+var filterfun = function(){
+  
+    if($(window).width() < 900)
+    {
+        $("#fil").toggleClass('hidefilter showfilter');
+        $("#sh").removeClass('showsortBy');
+        $("#sh").addClass('hidesortBy');
```

Comment:
use meaningful names for id and classes 
and convert above two lines into single one